module-types/Cisco/N77-SUP2E.yaml

```diff
@@ -0,0 +1,11 @@
+---
+manufacturer: Cisco
+model: N77-SUP2E
+part_number: N77-SUP2E
+interfaces:
+  - name: mgmt0
```

Comment:
This will cause a problem when inserting it into a chassis where there is two mgmt0 interfaces.  I believe it actually throws an exception.

Reply:
I added the following to my documents to address this issue.
interfaces:
  - name: Sup{module} mgmt0
    type: 1000base-t
    mgmt_only: true
console-ports:
  - name: Sup{module} Con0
    type: rj-45

Reply:
@scoffman77 The change needs to be made on the module, not the upstream